src/main/java/application/service/OrderItemService.java

```diff
@@ -0,0 +1,15 @@
+package application.service;
+
+import application.dto.item.OrderItemResponseDto;
+import application.model.OrderItem;
+import java.util.List;
+import org.springframework.security.core.Authentication;
+
+public interface OrderItemService {
+    OrderItem save(OrderItem orderItem);
+
+    List<OrderItemResponseDto> getAllOrderItemsDtoByOrderId(Authentication authentication,

```

Comment:
Actually not sure if is is useful to mention `authentication` in the service contract. First of all, we can call it from the corresponding service and not pass as an argument. But I guess the most preferable here would be to use some method level `@PreAuthorize` with custom permission check
https://docs.spring.io/spring-security/reference/servlet/authorization/method-security.html